### DISCUSSION TITLE
Unable to register any new users in the user profiles tabkle.

### DISCUSSION THREAD

**Discussion by mehtasdeepak** [supabase/supabase#30728] [Questions]
I am new user, just tyring to build my first web app 

Using bolt.new with supabase and i keep signing up as a new users and it keeps failing to add a record in user profiles table.

Although i can see the user created in auth table but i don't see them in user profile table.

You can watch video here.https://x.com/mehtasdeepak/status/1862340554391994823

<img width="1491" alt="Screenshot 2024-11-28 at 10 58 43 PM" src="https://github.com/user-attachments/assets/1c56dc19-3f3e-4412-8f2b-25a268696c5f">

<img width="1240" alt="Screenshot 2024-11-28 at 10 59 01 PM" src="https://github.com/user-attachments/assets/bd59b048-70ab-48db-92af-5977456d6a9d">

**Comment by silentworks** [comment]
When you hit these soft of issues you should first check you Auth logs as it gives more information as to what went wrong and also your postgres logs too.

**Comment by mehtasdeepak** [comment]
This is the auth log , looks like it needs supabase admin access , 

<img width="1336" alt="Screenshot 2024-11-29 at 10 40 21 AM" src="https://github.com/user-attachments/assets/47e908c4-5853-4ab7-a3ab-2bd556fc158f">

**Reply by GaryAustin1** [reply]
Whatever is calling the admin/users auth function in the client side needs to have the service_role key.  It probably has the anon key or a user session which will not work.
